packages/core/src/client/decorators/smartAccountClient.ts

```diff
@@ -54,19 +60,24 @@ export type BaseSmartAccountClientActions<
     args: SendUserOperationParameters<TAccount>
   ) => Promise<UserOperationStruct>;
   buildUserOperationFromTx: (
-    args: SendTransactionParameters<TChain, TAccount>
+    args: SendTransactionParameters<TChain, TAccount>,
+    overrides?: UserOperationOverrides
```

Comment:
ah wait, should this be like instead:

```suggestion
    args: SendTransactionParameters<TChain, TAccount> & {overrides?: UserOperationOverrides}
```

Reply:
refer to my comment below. @moldy530